### DISCUSSION TITLE
x-data has multiple objects, V3 exists?

### DISCUSSION THREAD

**Discussion by ericxin1982** [alpinejs/alpine#3912] [4. General]
![image](https://github.com/alpinejs/alpine/assets/9047436/82514e11-2398-47dc-aef7-779f641186f3)

That still exists?

Thanks

**Comment by ekwoka** [comment]
This isn't an alpine thing, just javascript, but yes it does work.

Just know that it will break getters/setters

**Reply by ericxin1982** [reply]
Got it. Thanks
